### PR TITLE
The 10th community highlight!

### PR DESCRIPTION
**Pull request by klaasnicolaas**
## Proposed change

A new community highlight blog! 😄 

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_posts/2021-02-26-community-highlights.markdown [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  />
+</a>
+
+## Lovelace dashboard inspiration
+
+Still looking for some new inspiration for your Lovelace dashboards? Last week we
+came across some beautiful Lovelace dashboards on our [Home Assistant subreddit][reddit]
+page of [swake88](https://www.reddit.com/user/swake88/). Who knows, there might be something fun for you to make.
+
+You can find [here](https://www.reddit.com/r/homeassistant/comments/lqo7wr/ha_2_years_later_and_what_ive_built_so_far/) the related reddit topic.
```
Comment (frenck) [inline source/_posts/2021-02-26-community-highlights.markdown]:
```suggestion
You can find [here](https://www.reddit.com/r/homeassistant/comments/lqo7wr/ha_2_years_later_and_what_ive_built_so_far/) the related Reddit topic.
```

File: source/_posts/2021-02-26-community-highlights.markdown [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+<blockquote class="imgur-embed-pub" lang="en" data-id="a/8BHxBVN"  ><a href="//imgur.com/a/8BHxBVN">Home Assistant - Feb 2021</a></blockquote><script async src="//s.imgur.com/min/embed.js" charset="utf-8"></script>
+
+Which card in lovelace are you most proud of? Share it with us!
```
Comment (frenck) [inline source/_posts/2021-02-26-community-highlights.markdown]:
```suggestion
Which card in Lovelace are you most proud of? Share it with us!
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by klaasnicolaas** [comment]
Ready for merge I think 😄

**Review by frenck — DISMISSED** [review]

File: source/_posts/2021-02-26-community-highlights.markdown [hunk 3]
```diff
@@ -0,0 +1,99 @@
+---
+title: "Community Highlights: 10th edition"
+description: "Try the new ApexCharts card, build your own switch panel and get inspiration for your own Lovelace dashboard."
+date: 2021-02-25 00:00:00
```
Comment (frenck) [inline source/_posts/2021-02-26-community-highlights.markdown]:
```suggestion
date: 2021-02-26 00:00:00
```

**Review by frenck — APPROVED** [review]

**Pull request merged**